packages/react-components/src/components/video/index.tsx

```diff
@@ -133,11 +133,8 @@ const Video = ({
         if (el) {
             const playhead = el.currentTime
             quartileEvents.some((q: QuartileEvent) => {
-                if (shouldFireQuartile(q, playhead, el.duration, quartilesFired.current, loopNumber.current)) {
-                    onQuartile?.(q)
-                    return true
-                }
-                return false
+                onQuartile?.(q)
```

Comment:
@matheusmorett2 we need the check here for `shouldFireQuartile` otherwise this is going to fire on every playhead update